docs/src/Model/HasCodeLinks.php

```diff
@@ -38,7 +38,7 @@ public function get_file_path() {
 	public function get_github_link() {
 		// TODO: Adapt based on which tag to use for documentation.
```

Comment:
@schlessera would this todo still be relevant with the change below? The link would be relevant to the branch or tag being browsed.

Reply:
No, this is not relevant anymore.